pyrobosim/pyrobosim/utils/general.py

```diff
@@ -112,8 +112,12 @@ def parse_color(color):
     :return: RGB tuple in range (0.0, 1.0).
     :rtype: tuple[float, float, float]
     """
-    if isinstance(color, (list, tuple)) and len(color) == 3:
-        return tuple(color)
+    if isinstance(color, (list, tuple)):
+        if len(color) == 3:
+            return tuple(color)
+        raise ValueError(
+            "Insufficient elements. RGB color must have exactly 3 elements."
```

Comment:
Can probably replace "Insufficient" with "incorrect number of". Since it could also be less than 3.